src/app/twinclass/page.tsx

```diff
@@ -60,15 +61,22 @@ export default function TwinClasses() {
     const tableRef = useRef<DataTableHandle>(null);
 
     async function fetchData(pagination: PaginationState, filters: FiltersState) {
-        const abstractFilter = filters?.filters["abstract"] === 'indeterminate' ?
-            undefined : filters?.filters["abstract"] as boolean;
         try {
             console.log('filters', filters)
             const {data, error} = await api.twinClass.search({
                 pagination,
                 search: filters?.search,
```

Comment:
TODO: remove this `search` field since it was replaced by `twinClassKeyLikeList` filter-field